service/index/writer.go

```diff
@@ -163,3 +163,26 @@ func (w *Writer) Events(height uint64, events []flow.Event) error {
 	}
 	return nil
 }
+
+// Seals indexes the seals, which should represent all seals in the finalized
+// block at the given height.
```

Comment:
WDYM by _should_? From what I understand it should be

```suggestion
// Seals indexes the seals from the finalized block at the given height.
```

But maybe I am missing something?

Reply:
You're right, but I followed suite with the style of the `Events` method comment.

Technically, one could call this function with *any* set of seals, but one *should* call it with the seals that were in the given block. At least that's my understanding why the style is as it is.